src/Domain/Campaign.php

```diff
@@ -30,6 +32,12 @@ class Campaign extends SalesforceReadProxy
      */
     protected Charity $charity;
 
+    /**
+     * @ORM\ManyToMany(targetEntity="CampaignFunding", mappedBy="campaigns")
+     * @psalm-suppress PossiblyUnusedProperty Used in Doctrine ORM mapping
+     */
+    protected Collection $campaignFundings;
```

Comment:
ah I just remembered since PHP 8.1 we can use [new in initialisers](https://stitcher.io/blog/php-81-new-in-initializers). So we can leave the constructor as it was and do

```suggestion
    protected Collection $campaignFundings = new ArrayCollection();
```

Reply:
Oops no that isn't allowed on a property ignore me.